### DISCUSSION TITLE
student package

### DISCUSSION THREAD

**Discussion by nvrzqsmv** [community/community#119851] [GitHub Education] [answered]
### Hi everyone,

Hello, I am Novruz Namig oglu Gasimov. I study at Baku State University. I am 28 years old. I have uploaded a photo of my student card. please confirm my subscription to the education package. Thank you in advance

**Comment by samus-aran** [comment, marked as answer]
Hi @nvrzqsmv 👋 

You can do this yourself by checking your status[ here ](https://education.github.com/discount_requests/pack_application). 👍

**Reply by nvrzqsmv** [reply]
I can't control my cravings

**Reply by nvrzqsmv** [reply]
I can't control my cravings
